core/src/test/java/org/elasticsearch/index/fielddata/BinaryDVFieldDataTests.java

```diff
@@ -56,24 +58,24 @@ public void testDocValue() throws Exception {
         bytesList1.add(randomBytes());
         bytesList1.add(randomBytes());
         XContentBuilder doc = XContentFactory.jsonBuilder().startObject().startArray("field").value(bytesList1.get(0)).value(bytesList1.get(1)).endArray().endObject();
-        ParsedDocument d = mapper.parse("test", "test", "1", doc.bytes());
+        ParsedDocument d = mapper.parse(SourceToParse.source("test", "test", "1", doc.bytes(),XContentType.JSON));
         writer.addDocument(d.rootDoc());
 
         byte[] bytes1 = randomBytes();
         doc = XContentFactory.jsonBuilder().startObject().field("field", bytes1).endObject();
-        d = mapper.parse("test", "test", "2", doc.bytes());
+        d = mapper.parse(SourceToParse.source("test", "test", "2", doc.bytes(),XContentType.JSON));
```

Comment:
Added a space between , and x for any of theses cases in all instances of the parse method.